<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>Virus</class>
    <widget class="QWidget" name="Virus">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>1000</width>
                <height>618</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Virus</string>
        </property>
        <property name="styleSheet">
            <string notr="true">QWidget#AntiVirus
                {
                background-color:white
                }
            </string>
        </property>
        <widget class="QPushButton" name="pushButton_start">
            <property name="enabled">
                <bool>true</bool>
            </property>
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>128</width>
                    <height>64</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_start
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_start:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_start:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>开始</string>
            </property>
        </widget>
        <widget class="QLabel" name="label_filepath">
            <property name="geometry">
                <rect>
                    <x>128</x>
                    <y>0</y>
                    <width>872</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="text">
                <string/>
            </property>
        </widget>
        <widget class="QProgressBar" name="progressBar">
            <property name="geometry">
                <rect>
                    <x>128</x>
                    <y>32</y>
                    <width>872</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="maximum">
                <number>0</number>
            </property>
            <property name="value">
                <number>0</number>
            </property>
            <property name="orientation">
                <enum>Qt::Horizontal</enum>
            </property>
            <property name="textDirection">
                <enum>QProgressBar::TopToBottom</enum>
            </property>
            <property name="format">
                <string>%p%</string>
            </property>
        </widget>
        <widget class="QListWidget" name="listWidget_program">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>96</y>
                    <width>1000</width>
                    <height>522</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QListWidget#listWidget_program
                    {
                    border:none
                    }
                </string>
            </property>
        </widget>
        <widget class="QLineEdit" name="lineEdit_way">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>64</y>
                    <width>936</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="text">
                <string>C:\</string>
            </property>
            <property name="placeholderText">
                <string>请输入扫描路径</string>
            </property>
        </widget>
        <widget class="QPushButton" name="pushButton_choose">
            <property name="geometry">
                <rect>
                    <x>936</x>
                    <y>64</y>
                    <width>64</width>
                    <height>32</height>
                </rect>
            </property>
            <property name="styleSheet">
                <string notr="true">QPushButton#pushButton_choose
                    {
                    border:0px solid rgb(16,124,16);
                    background-color:white
                    }
                    QPushButton#pushButton_choose:hover
                    {
                    border:2px solid rgb(16,124,16);
                    border-color:green
                    }
                    QPushButton#pushButton_choose:pressed
                    {
                    background-color:rgb(195, 195, 195)
                    }
                </string>
            </property>
            <property name="text">
                <string>...</string>
            </property>
        </widget>
    </widget>
    <resources/>
    <connections/>
</ui>
